src/OpenConext/ProfileBundle/Resources/config/services.yml

```diff
@@ -19,6 +19,17 @@ services:
         tags:
             - { name: twig.extension }
 
+    profile.global_view_parameters:
+        class: OpenConext\ProfileBundle\Service\GlobalViewParameters
+        factory: [OpenConext\ProfileBundle\Service\GlobalViewParameters, createForLocales]
```

Comment:
Because the validation of the configuration parameters is not really part of the instantiation of the class that uses them. In other configurations, Symfony2's Configuration class handles this.

Also, we are using a `locales` variable that is not used inside the class, but only for validation. Giving it as a constructor argument would imply it is significant to the inner workings of the class.